### PR TITLE
5.2.0 issue explicit expansion inheritance

### PR DESCRIPTION
**Pull request by moudrick**
#1958

### CODE REVIEW

File: src/AutoMapper/PropertyMap.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public PropertyMap(PropertyMap inheritedMappedProperty, TypeMap typeMap)
         public LambdaExpression CustomExpression { get; private set; }
         public MemberInfo CustomSourceMember { get; set; }
         public bool UseDestinationValue { get; set; }
-        public bool ExplicitExpansion { get; set; }
+        public bool ExplicitExpansion { get { return _explicitExpansion.GetValueOrDefault(); } set { _explicitExpansion = value; } } private bool? _explicitExpansion;
```
Comment (jbogard) [inline src/AutoMapper/PropertyMap.cs]: Should ExplicitExpansion just be nullable?

Comment (jbogard) [inline src/AutoMapper/PropertyMap.cs]: I say that because I converted other inheritable members to nullable

Comment (lbargaoanu) [inline src/AutoMapper/PropertyMap.cs]: But if Include doesn't really work with EF, how is this useful?

Comment (jbogard) [inline src/AutoMapper/PropertyMap.cs]: Just purely for inheriting configuration. Include works two ways - applying base config to children, and at runtime mapping doing the polymorphic stuff. I can see the configuration inheritance being valuable.

Comment (moudrick) [inline src/AutoMapper/PropertyMap.cs]: Anyway, it is confusing pretty much with or without working with EF. 

If inheritance can be done within some other scope, my issue could be added to that scope as a related issue. If this is going to be fixed in that scope and the code in my PR is not much helpful, the PR can be just declined, or I'll revoke it, if GitHub allows something like this, If the issue is found irrelevant, we can just close it, leaving this confuse intact.

Thanks, Oleksii.

Comment (jbogard) [inline src/AutoMapper/PropertyMap.cs]: Well I really just wanted my original question answered - should ExplicitExpansion just be nullable, and remove that backing field.

Comment (moudrick) [inline src/AutoMapper/PropertyMap.cs]: @jbogard, I think that ExplicitExpansion and other bool properties listed in the link below need to be non-nullable booeans, but backing fields have to be nullable for including base configuration sake.

The list of bool mapping config properties is @ SO answer by Ivan Stoev: http://stackoverflow.com/questions/41945809/automapper-5-2-ignores-explicitexpansion-if-it-is-configured-in-base-dto-mapping/42012290#42012290

The coding offer to introduce nullable backing field in below in the Ivan Stoev's comment to this answer: http://stackoverflow.com/questions/41945809/automapper-5-2-ignores-explicitexpansion-if-it-is-configured-in-base-dto-mapping/42012290#comment71390369_42012290

Comment (jbogard) [inline src/AutoMapper/PropertyMap.cs]: For consistency's sake, I'd make it a nullable property like all the others.

**Pull request closed without merging**